roles/transformers/vars/main.yml

```diff
@@ -1,8 +1,8 @@
 ---
 # vars file for transformers
-libreoffice_repo: "http://downloadarchive.documentfoundation.org/libreoffice/old"
-libreoffice_repo_path: "{{ dependencies_version.libreoffice }}/{{ pkg_format }}/{{ ansible_architecture }}"
-libreoffice_archive_name: /LibreOffice_{{ dependencies_version.libreoffice }}_{{ ansible_system }}_{{ ansible_architecture | regex_replace('_','-') }}_{{ pkg_format }}.tar.gz
+libreoffice_repo: "{{ artifacts_repositories.community.base_url }}/repositories/thirdparty/org/libreoffice/libreoffice-dist"
+libreoffice_repo_path: "{{ dependencies_version.libreoffice }}"
+libreoffice_archive_name: "libreoffice-dist-{{ dependencies_version.libreoffice }}-{{ pkg_format }}.gz"
 libreoffice_archive_url: "{{ libreoffice_repo }}/{{ libreoffice_repo_path }}/{{ libreoffice_archive_name }}"
 libreoffice_archive_sha1_checksum_url: "{{ libreoffice_archive_url }}.sha1"
```

Comment:
You don't this anymore then

Reply:
nexus still exposing it but in the format ansible expect, and for this reason I had to change https://github.com/Alfresco/alfresco-ansible-deployment/pull/463/files#diff-c4d0ad07c9ccf6e5893488fbf597089dc261c1aea39bfb3e0c69ebb7589d755fR14

Reply:
sorry I forgot a word: "You don't NEED this anymore"
I meant this var is not used anywhere now

Reply:
yes [it is](https://github.com/Alfresco/alfresco-ansible-deployment/pull/463/files#diff-c4d0ad07c9ccf6e5893488fbf597089dc261c1aea39bfb3e0c69ebb7589d755fR14)